diff --git a/bindings/python/lang.py b/bindings/python/lang.py
index bd7ef70..644de23 100644
--- a/bindings/python/lang.py
+++ b/bindings/python/lang.py
@@ -739,7 +739,6 @@ register_constants(PyObject *d)
             else:
                 print_('E: unknown constant type: %r' % c[0], file=sys.stderr)
             print_('    PyDict_SetItemString(d, "%s", obj);' % c[1][6:], file=fd)
-            print_('    Py_DECREF(obj);', file=fd)
         print_('}', file=fd)
         print_('', file=fd)
 
